README.md

```diff
@@ -15,7 +15,7 @@ npm install -g @elastic/synthetics
 Run the suites via CLI
 
 ```
-npx @elastic/synthetics [options] [dir] [files] file
+cd my-test-dir; npx @elastic/synthetics [options] [dir] [files] file
```

Comment:
We have in the documentation section links to running with heartbeat. Should we consider adding a link out to running with Fleet and Elastic Agent as well?